morebits.js

```diff
@@ -2601,34 +2601,15 @@ Morebits.wiki.page = function(pageName, currentAction) {
 			link.appendChild(document.createTextNode(ctx.pageName));
 			ctx.statusElement.info(['completed (', link, ')']);
 			if (ctx.onSaveSuccess) {
-				ctx.onSaveSuccess(this);  // invoke callback
+				ctx.onSaveSuccess(this);  // invoke 1callback
```

Comment:
`1callback`?

Reply:
Well, you see, since it's just "one callback" being invoked I figured... nah, I must've missed a meta key when switching buffers in emacs 😆